src/stream/mod.rs

```diff
@@ -6,5 +6,4 @@ pub mod types;
 pub mod video_stream_redirect;
 pub mod video_stream_rtsp;
 pub mod video_stream_udp;
-pub mod video_stream_webrtc;
 pub mod webrtc;
```

Comment:
why this was not removed ?

Reply:
And about anyhow ?

Reply:
I left anyhow because I am using it in our new WebRTC implementation, but I can remove and add again later if it makes more sense

Reply:
Ok, it's fine since it was explained. But usually removing and adding back is the cleaner solution, since we can't predict future code. 
We should also add a linter to notify about unused includes.